aboutus.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html class="page">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css">
```

Comment:
It would be better to put the link to your css file after the google font and animation library.